en/setup/fast-rtps-installation.md

```diff
@@ -20,10 +28,9 @@ Java is required to make use of our built-in code generation tool *fastrtpsgen*.
 
 *eProsima Fast RTPS* requires the Visual C++ Redistributable packages for the Visual Studio version you choose during the installation or compilation. The installer gives you the option of downloading and installing them.
```

Comment:
@VicenteEprosima @vilhjalmur89 "eProsima Fast RTPS* requires the Visual C++ Redistributable packages for the Visual Studio version you choose during the installation or compilation. "
This is confusing, because this is a requirement - at this point I haven't chosen or installed anything, so it isn't clear where I should have got the visual studio. Can you clarify?